src/components/Map/MapComponent.scss

```diff
@@ -0,0 +1,8 @@
+.map-container {
```

Comment:
Same here, remove this file and apply styles using `tailwind` (I left a comment with an example below),